race-buffer/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "race-buffer"
+version = "0.1.0"
+authors = ["Nicholas Springer <nspringer@auxon.io>"]
+edition = "2018"
+
+[dependencies]
+crossbeam = {version = "0.7.3", optional = true}
```

Comment:
Shouldn't `crossbeam` and `rand` be in `dev-dependencies` not regular dependencies?